examples/green-house-migration/routes/api/users.py

```
@@ -0,0 +1,34 @@
+
+@router.get("/user_rolis")
+def list_user_rolis(
+    pagination: Dict[str, Any] = Depends(pagination_dependency),
+):
+    """List all user rolis with pagination and metadata"""
+    return paginated_get("user_rolis", pagination)
+
+
+@router.get("/{user_id}/permissions/jobs")
+def get_user_job_permissions(
+    user_id: int, pagination: Dict[str, Any] = Depends(pagination_dependency)
```

Ruff (0.12.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)